### ISSUE TITLE
Missing information about GeneratePathProperty

### ISSUE THREAD

**Issue by rossng** [NuGet/docs.microsoft.com-nuget#1506]
This page should document the `GeneratePathProperty` attribute for the `PackageReference` element.

Related:
https://github.com/NuGet/Home/issues/4837#issuecomment-474184722
https://github.com/NuGet/NuGet.Client/pull/2271

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff707668-94d6-25a5-9139-b18cc1b6b446
* Version Independent ID: 494847b9-4894-0beb-7ffa-39a2359999fa
* Content: [NuGet PackageReference format (package references in project files)](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files)
* Content Source: [docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Package-References-in-Project-Files.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

**Comment by Mikejo5000** [comment]
@nkolev92 Per Jeff's comments in #7851, this feature is not intended for common use. Is it better to leave as an undocumented feature?

**Comment by nkolev92** [comment]
@rrelyea thoughts? 

My thinking is we doc it, but I'd wait on Rob's input for that.

**Comment by nkolev92** [comment]
Duplicate of https://github.com/NuGet/docs.microsoft.com-nuget/issues/1355
